metrics/metrics.go

```diff
@@ -76,6 +76,7 @@ type Metrics interface {
 	IncErrorsStreaming(routeId string)
 	RegisterHandler(path string, handler *http.ServeMux)
 	UpdateGauge(key string, value float64)
+	Close()
```

Comment:
I suggest to add a comment into commit and release message and increase minor version. I guess nobody implements this, so not really breaking but who knows. :)

Reply:
Will move metrics change into a separate PR.

Reply:
#2505